src/MusicStore/Get-Crossgen.ps1

```diff
@@ -32,9 +32,9 @@ function Get-CoreCLRVersion()
 
     foreach ($name in $json["libraries"].Keys)
```

Comment:
/cc @eerhardt @ericstj 

What's a reliable way to determine the CoreCLR version given a restored project? Right now we're looking in the assets file for `Microsoft.NETCore.Platforms` because that seems to match.